netcontrold/lib/dataif.py

```diff
@@ -25,6 +25,7 @@
 from netcontrold.lib import util
 
 from netcontrold.lib import config
+import operator
```

Comment:
why is this module included ?

Reply:
in upcall_rate calculation function.. operator.sub and operator.truediv are being used. For that purpose I included that